pycsw/core/util.py

```diff
@@ -33,11 +33,15 @@
 import time
 import datetime
 import logging
-import urllib2
+try:
+    from urllib2 import Request, urlopen
+except:
+    from urllib.request import Request, urlopen
 from shapely.wkt import loads
 from owslib.util import http_post
 from pycsw.core.etree import etree
 
+from six import text_type
```

Comment:
No problem.  We can workaround them or use a `six.py` copy with the parts we need.  (Six was actually intended to be used that way.)  Let me just work some `encode/decode` details out and them I figure out the best strategy.